docs/3. Verification/2. exactMatchVsMatch.md

```diff
@@ -19,24 +19,24 @@ Different compilation settings i.e. compiler versions, optimization runs result
 
 :::
 
-If the bytecode from recompiling the contract with the given source code files and the metadata correspond to the bytecode of the contract of the given address and chain, it will be a **match**. Sourcify defines two types of matches upon verifying contracts: **full (perfect) matches** and **partial matches**.
+If the bytecode from recompiling the contract with the given source code files and the metadata correspond to the bytecode of the contract of the given address and chain, it will be a **match**. Sourcify defines two types of matches upon verifying contracts: **Exact Matches** and **Matches**.
 
 :::info Info
 
-The match levels will be renamed in Sourcify's API v2:
+The match levels used a different terminology previously:
 
-- Full match will be renamed to **exact match**
-- Partial match will be renamed to **match**
+- Exact Match was called **Full / Perfect Match**
+- Match was called **Partial Match**
 
-This only concerns the naming, and the description of the match levels on this page will still be valid.
+This only concerns the naming. Their semantics were the same before.
 
 :::
 
-## Full (Perfect) Matches
+## Exact Matches
 
-<AvailableOnlyForSolidityAdmonition description="Vyper contracts don't support full matches because Vyper doesn't include the metadata hash in the bytecode."/>
+<AvailableOnlyForSolidityAdmonition description="Vyper contracts don't support exact matches because Vyper doesn't include the metadata hash in the bytecode."/>
 
-Full matches (sometimes referred as _perfect matches_) refer to the cases when the bytecode of the deployed contract is byte-by-byte the same as compilation output of the given source code files under the compilation settings defined in the [metadata file](/docs/metadata).
+Exact Matches refer to the cases when the bytecode of the deployed contract is byte-by-byte the same as compilation output of the given source code files under the compilation settings defined in the [metadata file](/docs/metadata).
```

Comment:
Looking again I think this is wrong. Immtuables, libraries etc. can be different. The byte-by-byte match is for the sources and the settings. Maybe we should mention that the integrity hashes or fingerprints of two compilations match.

Reply:
I revised the description a bit